src/index.html

```diff
@@ -1,14 +1,28 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Snake</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Snake</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Snake</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="container">
+      <div class="block block--1">1</div>
+
+      <div class="block block--2">2</div>
+
+      <div class="block block--3">3</div>
+
+      <div class="block block--4">4</div>
+
+      <div class="block block--5">5</div>
+
+      <div class="block block--6">6</div>
+    </div>
```

Comment:
```suggestion
    <div class="container">
      <div class="block block--1">1</div>
      <div class="block block--2">2</div>
      <div class="block block--3">3</div>
      <div class="block block--4">4</div>
      <div class="block block--5">5</div>
      <div class="block block--6">6</div>
    </div>
```

Reply:
I can't agree with you, @marina-tilniak:
![image](https://user-images.githubusercontent.com/30585774/168025079-e0600e1f-ecb0-4f98-9cf6-0f1f92cb033f.png)